lib/Mojolicious/Plugin/OpenAPI/Guides/OpenAPI3.pod

```diff
@@ -0,0 +1,392 @@
+=head1 NAME
+
+Mojolicious::Plugin::OpenAPI::Guides::OpenAPI3 - Mojolicious <3 Open API (Swagger)
```

Comment:
Should it be "OpenAPI v3" instead of "Open API" ?

Reply:
Would make sense to clarify it like that. (adding the change)